path_generator:
  planning_hz:
    type: double

  path_length:
    backward:
      type: double

    forward:
      type: double

  waypoint_group:
    separation_threshold:
      type: double

    interval_margin_ratio:
      type: double

  turn_signal:
    search_time:
      type: double

    search_distance:
      type: double

    resampling_interval:
      type: double

    angle_threshold_deg:
      type: double
